### ISSUE TITLE
Websockets support

### ISSUE THREAD

**Issue by rschildmeijer** [rschildmeijer/deft#16]

**Comment by rschildmeijer** [comment]
http://en.wikipedia.org/wiki/WebSockets
looks feasible

**Comment by jimpetersson** [comment]
Absolutely, however I think we should try to get the more "basic things" up and running before implementing this. Also it might be wise to do a proper design of it _before_ doing the actual implementation :)

**Comment by rschildmeijer** [comment]
websockets: http://www.jayway.com/media/3281/jayview21.pdf

**Comment by Bessonov** [comment]
Can I vote for this feature? :)

**Comment by rschildmeijer** [comment]
Absolutely. Using Apaches JIRA issue tracker since the incubation: https://issues.apache.org/jira/browse/DEFT
